### PR TITLE
add start of test suite

### PR DESCRIPTION
**Pull request by thijstriemstra**
refs #427 

Use `grunt`, `grunt test` or `grunt coverage` to run the tests. I added a single test for getId in util to show the setup is working.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -96,6 +96,33 @@ module.exports = function(grunt) {
         src: ['<%= concat.dist.src %>', 'plugin/*.js']
       },
     },
+    jasmine: {
+      core: {
+        src: '<%= concat.dist.src %>',
+        options: {
+          specs: 'spec/*Spec.js',
```
Comment (katspaugh) [inline Gruntfile.js]: Can we use `filename.spec.js` instead of camelCase?

Comment (thijstriemstra) [inline Gruntfile.js]: Sounds good. Will also try to get coveralls working on travis and rebase to a single commit when I'm done.

**Comment by thijstriemstra** [comment]
As you can see coverage works as well; https://coveralls.io/r/katspaugh/wavesurfer.js

**Comment by thijstriemstra** [comment]
@katspaugh, ready for review.

**Comment by katspaugh** [comment]
Cool, thanks!

**Comment by katspaugh** [comment]
Hey @thijstriemstra!

I've received a license key for WebStorm from JetBrains for the wavesurfer.js team, would you like to have one?

- K

**Pull request merged**